src/endpoints/authentication/jwt_auth.cpp

```diff
@@ -120,13 +120,12 @@ namespace ccf
     std::string& error_reason)
   {
     const auto& headers = ctx->get_request_headers();
+    error_reason = "Invalid JWT token";
```

Comment:
```suggestion
    error_reason = "Invalid JWT token";
```

Let's not set the error reason before we know there is an error, this may surprise the caller unncessarily.